calico_node/filesystem/etc/rc.local

```diff
@@ -27,12 +27,22 @@ else
 
     # Run confd twice.  Our confd TOML files are also generated from confd, so
     # running twice ensures our starting configuration is correct.
-    confd -confdir=/etc/calico/confd -onetime -node=${ETCD_SCHEME:=http}://${ETCD_AUTHORITY} \
+    case "$ETCD_ENDPOINTS" in
```

Comment:
IIRC, the `-scheme` was necessary for passing in to confd.  If you make this change, be sure to test it.  I'd expect it to fail.